cryptostock/account/tests/test_create_salesdashboard.py

```diff
@@ -0,0 +1,67 @@
+import decimal
+
+from account.tests.conftest import AssetFactory, WalletRecordFactory
+
+
+def test_valid_create_sales_dashboard(auth_broker, broker_account):
+    wallet_record = WalletRecordFactory(wallet=broker_account.wallet)
+    asset = wallet_record.asset
+    asset_count_before_create = broker_account.wallet.wallet_record.get(
+        asset=asset
+    ).count
+    data = {"asset": asset.id, "count": 100.4444, "price": 500.555555}
```

Comment:
Should `count` and `price` be decimal?
```suggestion
    data = {"asset": asset.id, "count": "100.4444", "price": "500.555555"}
```

Reply:
got it